src/lib/crypt.test.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @jest-environment node
+ */
+
+import { hash } from './crypt';
+
+describe('Crypt', () => {
+  describe('hash()', () => {
+    it('should generate hash for the string', async () => {
+      const str = 'test';
```

Comment:
Avoid abbreviated variable names (see: https://nimblehq.co/compass/development/code-conventions/#convention-12) and naming after object types (see: https://nimblehq.co/compass/development/code-conventions/#convention-16):

```suggestion
      const inputText = 'test';
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/44/commits/772649044e477fe0425328fbaae9c5acd0eccb73